careers.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
+  <title>Infratab Bangalore Pvt. Ltd</title>
+  <!-- CSS  -->
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+<nav class="navbar">
+  <div><a id="logo-container" href="#" class="brand-logo"><img src= "./img/infratab-white-logo.png"></a>
+    <ul class="right hide-on-med-and-down">
+      <li><a href="index.html">HOME</a></li>
+      <li><a href="careers.html">CAREERS</a></li>
+      <li><a href="contact.html">CONTACT</a></li>
+      <li><a href="www.infratab.com" target="_blank"><i class="fa fa-globe"></i></a></li>
+    </ul>
+    <ul id="nav-mobile" class="side-nav">
+      <li><a href="index.html">HOME</a></li>
+      <li><a href="careers.html">CAREERS</a></li>
+      <li><a href="contact.html">CONTACT</a></li>
+      <li><a href="www.infratab.com"><i class="fa fa-globe"></i></a></li>
+    </ul>
+    <a href="#" data-activates="nav-mobile" class="button-collapse"><i class="material-icons">MENUs</i></a>
+  </div>
+</nav>
+<section class="team scroll">
+  <div class="team-image"></div>
+  <div class="team-overlay"></div>
+  <div class="team-center">
+      <p class="team-caption">BUILD future with us</p>
+      <a class="team-button" id="js-join-scroll">Join us</a>
```

Comment:
@priyanka0901 From the design you know that this raised button then it should be written in `button` tag, not `a` tag. `a` tag should be used only for linking and if you want to make the flat button.